docker-compose.yaml

```diff
@@ -168,6 +172,26 @@ services:
     ports:
       - 80:80
 
+  mlflow:
+    image: ghcr.io/mlflow/mlflow:v2.0.1
+    environment:
+      - MLFLOW_TRACKING_URI=${MLFLOW_TRACKING_URI}
+      - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
+      - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
+      - AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION}
+      - BACKEND_STORE_URI=sqlite:///mlflow.db
+      - ARTIFACT_ROOT=s3://mlflow`
+    ports:
+      - "8001:5000"
```

Comment:
I'd actually move away from the well known range of 8xxx... Why not keep it at 5000?

Reply:
on Macos the Airplay server uses port 5000 😢  https://nono.ma/port-5000-used-by-control-center-in-macos-controlce

Our backend is exposed to port 8000 so I figured I would keep it close to that port.

Reply:
It's not a big deal. But consider why Ray uses 8265 as port 🤣